rust/src/connection/connection.rs

```diff
@@ -248,22 +323,20 @@ impl fmt::Debug for Connection {
 
 #[derive(Clone)]
 pub(crate) struct ServerConnection {
-    address: Address,
```

Comment:
The only reason for using this was to allow re-fetching the same `ServerConnection` from the overarching `Connection` later. In most cases, the address / server_id is already available.